src/TodoList.tsx

```diff
@@ -0,0 +1,106 @@
+import classNames from 'classnames';
+import { Todo } from './types/Todo';
+
+interface TodoListProps {
+  todos: Todo[],
+  removeTodo: React.Dispatch<React.SetStateAction<boolean>>,
+  editTodo: React.Dispatch<React.SetStateAction<boolean>>,
+}
+
+export const TodoList: React.FC<TodoListProps> = ({
+  todos,
+  removeTodo,
+  editTodo,
+}) => {
+  const handleTodoRemoval = () => {
+    removeTodo(true);
+  };
+
+  const handleTodoTitleChange = () => {
```

Comment:
handleTodoTitleChange should receive the todo id as an argument and pass it to editTodo

Reply:
@lerastarynets not sure i understand why i need to pass the todo.id to setEditTodoIsClicked


first of all, setEditTodoIsClicked is boolean

at this stage, I only wanted it to show the error on this stage.

please elaborate on it.